build.gradle

```diff
@@ -39,20 +39,23 @@ repositories {
 
 compileJava {
 	// please also update accordingly the animalsniffer config down below when changing the compatibility settings
-	sourceCompatibility = 1.6
-	targetCompatibility = 1.6
+	sourceCompatibility = 1.7
+	targetCompatibility = 1.7
 }
 
 compileTestJava {
-	sourceCompatibility = 1.6
-	targetCompatibility = 1.6
+	sourceCompatibility = 1.7
+	targetCompatibility = 1.7
```

Comment:
I am using `Throwable.addSuppressed(Throwable)` which was added in 1.7.

Note that JDK 6 reached EOL in Feb of 2013 according to http://www.oracle.com/technetwork/java/eol-135779.html (even 1.7 is past EOL).

If this is a concern, then we could move the `TestFixture` code to another library.